docs/C2D.md

```diff
@@ -56,7 +56,10 @@ These internal properties can be used to interact with any aspect of the Web's [
 | `height` | number | the [height](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/height) of the canvas, an alias for `C2D.canvas.height` |
 | `fill` | string | the current [fill style](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillStyle) of the drawing context, an alias for `C2D.ctx.fillStyle` |
 | `stroke` | string | the current [stroke style](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/strokeStyle) of the drawing context, an alias for `C2D.ctx.strokeStyle` |
-
+| `fontSize` | number | the font size in pixels of any font drawn on the canvas. Allows easier writing to [`C2D.ctx.font`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/font) Defaults to 48. |
```

Comment:
I think it might be a good idea to create a new column in the properties table for "default" (so that we can make the descriptions a bit shorter && defaults easier to identify at a glance)